babel.config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "presets": [
+    "@babel/preset-env",
+    "@babel/preset-react"
+  ],
+  "plugins": [
+    "add-module-exports",
+    "syntax-async-functions",
+    "@babel/plugin-syntax-dynamic-import",
```

Comment:
it looks like the migration tool may have done this for us, but out of curiosity, do you know why we needed to add the dynamic import plugin?

Reply:
Oh, actually I don't know if we do, I was just making it the same as code-dot-org repo while I was debugging stuff. I will verify if we need this before I commit it.